.github/workflows/bindings_ci.yml

```diff
@@ -0,0 +1,115 @@
+name: Bindings tests
+
+on:
+  workflow_dispatch:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+    types:
+      - opened
+      - reopened
+      - synchronize
+      - ready_for_review
+
+env:
+  CARGO_TERM_COLOR: always
+  MATRIX_SDK_CRYPTO_NODEJS_PATH: bindings/matrix-sdk-crypto-nodejs
+  MATRIX_SDK_CRYPTO_JS_PATH: bindings/matrix-sdk-crypto-js
```

Comment:
Why put this in environment variables?

Reply:
To not repeat myself. Now that I'm thinking of it, we may want to declare variables, but not in the global environment… I'm checking the [documentation](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions) and I don't see another way to declare variables. Did I miss something?

Reply:
I'm repeating these paths in several places, that's why I'm using a variable. But I agree that the size is not an argument here.